api/users/models.py

```diff
@@ -91,12 +98,33 @@ def get_by_natural_key(self, email):
         return self.get(email__iexact=email)
 
 
-class FFAdminUser(LifecycleModel, AbstractUser):
+# NOTE date_joined cannot be excluded because there is also a date_joined field on
+# the organisations through model, and currently simple history would create a
+# historical through model without the field but try to store the value anyway
```

Comment:
Is there a path to resolve this?

Reply:
Raise a proper ticket with django-simple-history I guess. I did think I'd fixed it at one point by patching it to use `tracked_fields` when creating records but I had not. I think for some reason the m2m `tracked_fields` doesn't reflect `exclude_fields` though the m2m model does.